backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -47,51 +49,64 @@
   private static final String KEY_PARTICIPANT = "Participant";
   private static final String KEY_MATCH = "Match";
   private static final String PROPERTY_USERNAME = "username";
-  private static final String PROPERTY_STARTTIMEAVAILABLE = "startTimeAvailable";
-  private static final String PROPERTY_ENDTIMEAVAILABLE = "endTimeAvailable";
+  private static final String PROPERTY_START_TIME_AVAILABLE = "startTimeAvailable";
+  private static final String PROPERTY_END_TIME_AVAILABLE = "endTimeAvailable";
   private static final String PROPERTY_DURATION = "duration";
   private static final String PROPERTY_TIMESTAMP = "timestamp";
-  private static final String PROPERTY_FIRSTPARTICIPANT = "firstParticipant";
-  private static final String PROPERTY_SECONDPARTICIPANT = "secondParticipant";
+  private static final String PROPERTY_FIRST_PARTICIPANT = "firstParticipant";
+  private static final String PROPERTY_SECOND_PARTICIPANT = "secondParticipant";
+
+  // HTTP Request JSON key constants
+  private static final String REQUEST_FORM_DETAILS = "formDetails";
+  private static final String REQUEST_TIME_AVAILABLE_UNTIL = "timeAvailableUntil";
+  private static final String REQUEST_DURATION = "duration";
+  private static final String REQUEST_ROLE = "role";
+  private static final String REQUEST_PRODUCT_AREA = "productArea";
+  private static final String REQUEST_SAVE_PREFERENCE = "savePreference";
+  private static final String REQUEST_MATCH_PREFERENCE = "matchPreference";
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Doesn't need to be in this PR but please file an issue to unit test this method.